wazo_agid/modules/call_recording.py

```diff
@@ -9,24 +9,34 @@
 
 
 def call_recording(agi, cursor, args):
+    calld = agi.config['calld']['client']
+    channel_id = agi.env['agi_uniqueid']
     if agi.get_variable('WAZO_CALL_RECORD_ACTIVE') == '1':
-        _disable_call_recording(agi, cursor, args)
+        _disable_call_recording(agi, calld, channel_id)
     else:
-        _enable_call_recording(agi, cursor, args)
+        _enable_call_recording(agi, calld, channel_id)
 
 
-def _enable_call_recording(agi, cursor, args):
+def _enable_call_recording(agi, calld, channel_id):
     record_filename = agi.get_variable('XIVO_CALLRECORDFILE')
     if not record_filename:
         logger.error('Could not start recording: could not determine record file')
         return
 
-    agi.appexec('MixMonitor', record_filename)
+    try:
+        calld.calls.start_record(channel_id)
+    except Exception as e:
+        logger.error('Error during enabling call recording: %s', e)
+
     agi.set_variable('WAZO_CALL_RECORD_ACTIVE', '1')
```

Comment:
This is now set by the API. Also for `_disable_call_recording`

Reply:
True, fixed (logic was done before api :sweat: , that's why I don't like starting ticket before other)